cdc/redo/common/redo.go

```diff
@@ -52,7 +59,59 @@ const (
 
 // LogMeta is used for store meta info.
 type LogMeta struct {
-	CheckPointTs   uint64           `msg:"checkPointTs"`
-	ResolvedTs     uint64           `msg:"resolvedTs"`
-	ResolvedTsList map[int64]uint64 `msg:"-"`
+	CheckPointTs   uint64
```

Comment:
Why remove the `msg` tag?

Reply:
Because now marsual and unmarshal are customed instead of generated.